internal/handlers/irc/irc.go

```diff
@@ -1,7 +1,8 @@
 package irc
 
 import (
-	"fmt"
+	"log"
+	"io/ioutil"
```

Comment:
Unused import here, as well as the other files that import this.

Reply:
Thanks for the comments. I have considered Tim's and your feedback to reorganise. I should have something for today's meeting.